deployment/common/changeset/mcms_helpers.go

```diff
@@ -0,0 +1,177 @@
+package changeset
```

Comment:
maybe belongs in the propose dir? Or can kill that dir and move it into mcms_helpers.go